### ISSUE TITLE
Like your live example, how could I force user to open a login page? 

### ISSUE THREAD

**Issue by avatar-lavventura** [lizkrznarich/orcid-demo-app#13]
This is not issue but question.

The way I started the php script: `php -S localhost:8000 index.php`

On the demo source code: when I click to `Already have an ORCID iD?` html tries to opens following url: [](https://orcid.org/oauth/authorize?client_id=APP-XXXXXXXXXXXXXXXX&response_type=code&scope=/authenticate&redirect_uri=https://your-redi\
rect-uri.org&show_login=true)

Later when I replaced client_id and redirect_uri that are embedded in the code; user's providedweb site opened and I couldn't see the `Thanks,` site. So it directs me into <http://users_website/?code=KAUVn0>.  But on the example demo (https://orcid-create-on-demand.herokuapp.com/) , I am directed into 'oauth-redirect.php'.

Basically how I can jump from index.php to oauth-redirect.php via obtaining `$_GET['code']`.

**[Q]** How is this possible?

**Comment by alainna** [comment]
the redirect_uri is the page that the user will be directed to after authorizing the system to get the ORCID iD. For example see the demo system https://orcid-create-on-demand.herokuapp.com 

The authorization call is: https://sandbox.orcid.org/oauth/authorize?client_id=APP-OTUM954K2JXQDK9Z&response_type=code&scope=/authenticate&show_login=false&redirect_uri=https://orcid-create-on-demand.herokuapp.com/oauth-redirect.php 

After the user grants permission, they are directed to https://orcid-create-on-demand.herokuapp.com/oauth-redirect.php with ?code=XXXXXX at the end. You must change the redirect URI to be the location in your system where the user will go.

**Comment by avatar-lavventura** [comment]
I have done your suggestion and it works. Please try on here (http://35.231.95.41/index.php), it does return and empty information page.

I am using `oauth-redirect.php` and I have updated:

```
define('OAUTH_CLIENT_ID',    
define('OAUTH_CLIENT_SECRET', 
define('OAUTH_REDIRECT_URI', 
```

I am not sure why `echo` for each parameter returns null.
 
<img width="1007" alt="screen shot 2018-05-16 at 18 54 07" src="https://user-images.githubusercontent.com/18537398/40128584-a3b3bd04-593a-11e8-9098-476f5c64550a.png">

Please note that I am running:  `php -S localhost:8000 oauth-redirect-idp.php`
But http://35.231.95.41:8000/oauth-redirect-idp.php does not open. I am not sure it is correct way to do this.
